lib/apiservers/engine/backends/container.go

```diff
@@ -69,6 +70,13 @@ const (
 	MinCPUs = 1
 	// DefaultCPUs - the default number of container VM CPUs
 	DefaultCPUs = 2
+
+	// Random numbers generated by fair dice roll :)
+	DefaultInitialInterval     = 2 * time.Second
```

Comment:
what is the minimum that this results in? Ideally we'd retry immediately on the first failure as our most common scenario is that there are only two operations competing (attach and start) and having received ConflictError once means that the competing operation has completed.